controller/studentController.js

```diff
@@ -0,0 +1,136 @@
+const Student = require('../models').Student
+
+class studentController {
+    static show(req, res) {
+        // console.log(Student)
+        Student.findAll()
+            .then(student => {
+                res.render('student', { student })
+            })
+            .catch(err => {
+                res.redirect('/*')
+            })
+    }
+
+    static addGet(req, res) {
+        const error = req.query.error;
+        res.render('addForm', { error })
+    }
+
+    static addPost(req, res) {
+        // console.log(req.body.birth_date)
+        const error = studentController.validation(req.body)
+        if (error.length > 0) {
+            res.redirect(`/student/add?error=${error.join(', ')}`)
+        } else {
+            Student.create({
+                first_name: req.body.first_name,
+                last_name: req.body.last_name,
+                email: req.body.email,
+                gender: req.body.gender,
+                birth_date: req.body.birth_date
+            })
+                .then(data => {
+                    res.redirect('/student')
+                })
+                .catch(err => {
+                    res.redirect('/*')
```

Comment:
error nya disampaikan ke client
```suggestion
                    res.render('<halaman apa>', { error: err }) // atau res.send(err)
```